### DISCUSSION TITLE
PMTiles raster overlay

### DISCUSSION THREAD

**Discussion by vitoborneo** [Leaflet/Leaflet#9251] [Q&A]
I'm looking for an example like this:
https://leafletjs.com/examples/layers-control/
To use a raster PMTiles as overlay and be able to choose the basemaps.
I get an error message "The provided object is not a Layer"
Can you help me?
Vito

**Comment by Falke-Design** [comment]
Hi, great to hear that you find Leaflet useful!

However, this issue / discussion tracker is used for reporting bugs and discussing new features. For questions on using Leaflet, please use [gis.stackexchange.com](https://gis.stackexchange.com/questions/tagged/leaflet) or [stackoverflow](https://stackoverflow.com/questions/tagged/leaflet).
